src/metadata/ro.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: ontology_detail
+id: ro
+title: OBO Relations Ontology
+jobs:
+  - id: https://travis-ci.org/oborel/obo-relations
```

Comment:
Travis-CI seems a bit outdated, no?

Reply:
Please Charlie. Please. Do not look at people PR drafts.. this is like checking someone out while they are getting dressed.